### PR TITLE
OpenAI steam API

### PR DESCRIPTION
**Pull request by jakule**

### CODE REVIEW

File: ai-service/ai_service/app.py [hunk 1]
```diff
@@ -1,29 +1,62 @@
...
+            verbose=True,  # Some BS, but when verbose is false, then the handler is not being called!!!!!
+        )
+
+    def Complete(self, request, context):
+        logging.debug("Complete called")
+
+        yield from assistant_query(self, request)
+
+
 def assistant_query(
```
Comment (xacrimon) [inline ai-service/ai_service/app.py:53]: Is there no way to use eg asyncio here? Would make this code about a *lot* easier to reason about.

**Comment by xacrimon** [comment]
Overall can we wait with these changes until I restructure some of the model in the service? That directly impacts this and in the current somewhat hacky state of this it seems unfortunate to merge if I need to keep working around these hacks to make streaming work when I have to reason about my changes.

File: ai-service/ai_service/app.py (hunk 1 above)
Comment (jakule) [reply ai-service/ai_service/app.py:53]: I think it should be possible.

**Comment by jakule** [comment]
> Overall can we wait with these changes until I restructure some of the model in the service? That directly impacts this and in the current somewhat hacky state of this it seems unfortunate to merge if I need to keep working around these hacks to make streaming work when I have to reason about my changes.

Sure, I think there are a few more problems that we need to solve anyways before this PR can be merged.

**Comment by jakule** [comment]
Closing this PR since we moved our implementation to Go.

**Pull request closed without merging**